dvc/scm/git/__init__.py

```diff
@@ -260,15 +260,12 @@ def get_tree(self, rev):
 
     def _get_diff_trees(self, a_ref, b_ref):
         """Private method for getting the trees and commit hashes of 2 git
-        references.
-        Requires `gitdb` module (from gitpython package).
+        references. Requires `gitdb` module (from gitpython package).
 
-        Args:
-            a_ref(str) - git reference
-            b_ref(str) - second git reference. If None, uses HEAD
+        Args: a_ref(str) - git reference
+              b_ref(str) - second git reference. If None, uses HEAD
 
-        Returns:
-            tuple - tuple with elements: (trees, commits)
+        Returns: tuple - tuple with elements: (trees, commits)
```

Comment:
and this one ^

Reply:
Fixed

Reply:
Nope, @jorgeorpinel , issue is still there.

Reply:
🤦‍♂ yeah actually and I was using `-` instead of `:` sry

Fixed this time 🤞